### PR TITLE
RHINENG-5922 support multiple kafka brokers

### PR DESCRIPTION
**Pull request by r14chandra**

### CODE REVIEW

File: yuptoo/lib/config.py [hunk 1]
```diff
@@ -22,9 +22,9 @@ def kafka_auth_config(connection_object):
 
 if CLOWDER_ENABLED:
     LOG.info("Using Clowder Operator...")
-    from app_common_python import LoadedConfig, KafkaTopics
+    from app_common_python import LoadedConfig, KafkaTopics, KafkaServers
     KAFKA_BROKER = LoadedConfig.kafka.brokers[0]
-    INSIGHTS_KAFKA_ADDRESS = KAFKA_BROKER.hostname + ":" + str(KAFKA_BROKER.port)
+    INSIGHTS_KAFKA_ADDRESS = KafkaServers
```
[the remark this answers is not in the document]
Comment (kgaikwad) [reply yuptoo/lib/config.py:27]: Also, handles it in `else` loop.

**Review by kgaikwad — CHANGES_REQUESTED** [review]
@r14chandra , just a small suggestion added on PR. 
As it includes multiple constant variables when it comes to local development. Could you update it?

File: yuptoo/lib/config.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (r14chandra) [reply yuptoo/lib/config.py]: done.

**Review by kgaikwad — APPROVED** [review]
LGTM :+1:  Thank you @r14chandra! Merging it.

**Pull request merged**